### ISSUE TITLE
No `documentElement` in XML documents

### ISSUE THREAD

**Issue by andrienko** [zeligzhou/xmldom-qsa#2]
I am parsing an XML document (not an HTML document). It works fine with the original `xmldom` (but it does not have a qsa, obviously).

this package gives following error:

```
TypeError: Cannot read properties of undefined (reading 'documentElement')
    at helper (/Users/lowpolybrain/js/mesh/common/temp/node_modules/.pnpm/xmldom-qsa@1.1.0/node_modules/xmldom-qsa/lib/helper.js:3:20)
    at Document.Class.querySelector (/Users/lowpolybrain/js/mesh/common/temp/node_modules/.pnpm/xmldom-qsa@1.1.0/node_modules/xmldom-qsa/lib/dom.js:1529:10)
    at null.setNode (/Users/lowpolybrain/js/mesh/packages/compiler/src/postProcessColladaXML.ts:18:28)
    at null.postProcessColladaXML (/Users/lowpolybrain/js/mesh/packages/compiler/src/postProcessColladaXML.ts:44:3)
    at DanketsuCompiler.<anonymous> (/Users/lowpolybrain/js/mesh/packages/compiler/src/index.ts:38:14)
    at null.step (/Users/lowpolybrain/js/mesh/node_modules/tslib/tslib.es6.js:100:23)
    at Object.next (/Users/lowpolybrain/js/mesh/node_modules/tslib/tslib.es6.js:81:53)
    at null.<anonymous> (/Users/lowpolybrain/js/mesh/node_modules/tslib/tslib.es6.js:74:71)
    at new Promise (<anonymous>)
    at null.__awaiter (/Users/lowpolybrain/js/mesh/node_modules/tslib/tslib.es6.js:70:12)
```
which comes from
```
function helper(g) {
  var doc = g.document,
  root = doc.documentElement,
```

Apparently, this would work fine if I was working with an HTML document, but does not work for XML

Example of an XML document I am catching the exception with: https://gist.github.com/andrienko/31a93ba884d6c93dae383bf0eb942bd6

And what I am roughly trying to do is
```
  const parser = new DOMParser();
  const xmlDocument = parser.parseFromString(colladaXML, 'text/xml');
  const contributor = xmlDocument.querySelector('asset contributor')
```

This works just fine with browser's DOMParser.

Just `xmldom` works fine here (but does not have a QSA, hehe)

**Comment by zeligzhou** [comment]
I'm sorry for the late reply.
 I tried to reproduce your scenario with the XML document you applied, but there was no problem with the running results. 
```
  let testString = await fs.readFileSync(__dirname + '/test.xml', "utf8")

  const parser = new DOMParser();
  const xmlDocument = parser.parseFromString(testString, 'text/xml');
  const contributor = xmlDocument.querySelector('asset contributor')
```

Can you provide more information, such as the node version...🙏

**Comment by ammarahm-ed** [comment]
@zeligzhou I am getting the same issue when parsing.

```js
import { DOMParser, XMLSerializer } from "xmldom-qsa";
const RealDOMParser = new DOMParser();
const RealDOMSerializer = new XMLSerializer();
const parseHTML = (input) => RealDOMParser.parseFromString(input, "text/html");
const serializeHTML = (input) => RealDOMSerializer.serializeToString(input);
globalThis.parseHTML = parseHTML;
globalThis.serializeHTML = serializeHTML;
  
const html = globalThis.parseHTML("<p></p>");
      console.log(html.getElementsByTagName("p").length); //works
      console.log(!!html.querySelector("p")); // does not work.
      return true
```

Error: `Error: Cannot read property 'documentElement' of undefined`

I am running this in `react-native` which is browser-like environment.

**Comment by zeligzhou** [comment]
@andrienko @ammarahm-ed Thank you for your report. I fix it in version 1.1.2, it works well in my local React Native project. Look forward to your kind feedback.😊

**Comment by ammarahm-ed** [comment]
Thanks for the quick fix! i will try it out.
